libs/lb/lb-rs/src/lib.rs

```diff
@@ -173,18 +173,42 @@ impl<Client: Requester, Docs: DocumentService> CoreLib<Client, Docs> {
             ])
     }
 
+    #[instrument(level = "debug", skip_all, err(Debug))]
+    pub fn import_account_v2(&self, phrases: [String; 24], api_url: &str) -> LbResult<Account> {
+        self.in_tx(|s| s.import_account_v2(phrases, api_url))
+            .expected_errs(&[
+                CoreError::AccountExists,
+                CoreError::AccountNonexistent,
+                CoreError::KeyPhrasesMistyped,
+                CoreError::ServerUnreachable,
+                CoreError::ClientUpdateRequired,
+            ])
+    }
+
     #[instrument(level = "debug", skip_all, err(Debug))]
     pub fn export_account(&self) -> Result<String, LbError> {
         self.in_tx(|s| s.export_account())
             .expected_errs(&[CoreError::AccountNonexistent])
     }
 
+    #[instrument(level = "debug", skip_all, err(Debug))]
+    pub fn export_account_v2(&self) -> Result<[String; 24], LbError> {
+        self.in_tx(|s| s.export_account_v2())
+            .expected_errs(&[CoreError::AccountNonexistent])
+    }
+
     #[instrument(level = "debug", skip_all, err(Debug))]
     pub fn export_account_qr(&self) -> Result<Vec<u8>, LbError> {
         self.in_tx(|s| s.export_account_qr())
             .expected_errs(&[CoreError::AccountNonexistent])
     }
 
+    #[instrument(level = "debug", skip_all, err(Debug))]
+    pub fn export_account_qr_v2(&self) -> Result<Vec<u8>, LbError> {
```

Comment:
imo our qr code should just be base64 repr of the key and we're just going to support both perpetually